tests/test_git_init.py

```diff
@@ -5,46 +5,28 @@
 
 import pytest
 
+from .helpers import gen_package
+
 
 @pytest.mark.parametrize(
     "git_init_cookiecutter_option",
-    ["initialise_git_repository=True", "initialise_git_repository=False"],
+    ["True", "False"],
```

Comment:
I think, as a minimum, they don't need to be strings
```suggestion
    [True, False],
```
so can also change `if "True" in git_init_cookiecutter_option` to `if git_init_cookiecutter_option`

Reply:
You can pass in a one-element dictionary and expand it when you define the config dictionary...

```python
@pytest.mark.parametrize(
    "one_element_dictionary",
    [{"initialise_git_repo": False}, {"initialise_git_repo": True}],
)
def test_append_to_dictionary(one_element_dictionary):
    config = {
        "github_username": "test-user",
        **one_element_dictionary,
    }

```

Reply:
Since I'm the one taking over, I did this. @paddyroddy - is [017b8f4](https://github.com/UCL-ARC/python-tooling/pull/405/commits/017b8f4b6d0d92ac55b15de684db5b2bfd23c570) more readable?

Reply:
Yes sir